### ISSUE TITLE
onNotificationRemoval not getting called

### ISSUE THREAD

**Issue by GabrielBB** [teodosii/react-notifications-component#29]
I want to call my function when user clicks the notification. I tried `onNotificationRemoval `but is not working. Please, if you know some way i can achieve this, let me know asap :/

 ```
store.addNotification({
          title: message.firstName + " is inviting you to play",
          message: "Click here to accept",
          type: "default",
          insert: "top",
          width: 400,
          container: "top-right",
          animationIn: ["animated", "fadeIn"],
          animationOut: ["animated", "fadeOut"],
          onNotificationRemoval: () => this.onAcceptInvitation(message)
        });
```

![image](https://user-images.githubusercontent.com/8752823/64836440-eb01b580-d5b7-11e9-807c-ec37374c5ed4.png)

**Comment by teodosii** [comment]
Fixed in `2.1.0` - see changelog/readme

**Comment by GabrielBB** [comment]
@teodosii Great! Thanks
